tests/native_tests/with_test.hy

```diff
@@ -34,12 +34,37 @@
         (assert (= t2 2))
         (assert (= t3 3))))
 
-  (defn test-quince-with []
-    "NATIVE: test four withs, one with no args"
-    (with [t1 (WithTest 1)
-           t2 (WithTest 2)
-           t3 (WithTest 3)
-           _ (WithTest 4)]
-          (assert (= t1 1))
-          (assert (= t2 2))
-          (assert (= t3 3))))
+(defn test-quince-with []
+  "NATIVE: test four withs, one with no args"
+  (with [t1 (WithTest 1)
+         t2 (WithTest 2)
+         t3 (WithTest 3)
+         _ (WithTest 4)]
+        (assert (= t1 1))
+        (assert (= t2 2))
+        (assert (= t3 3))))
+
+(defclass SuppressZDE [object]
+  (defn --enter-- [self])
+  (defn --exit-- [self exc-type exc-value traceback]
+    (and (not (none? exc-type)) (issubclass exc-type ZeroDivisionError))))
+
+(defn test-exception-suppressing-with []
+  ; https://github.com/hylang/hy/issues/1320
```

Comment:
Real talk. I sincerely appreciate how you've resumed work on Hy. You're working on a lot of cool PRs and you're often helpful. But, I don't think it's a good use of my time, or your time, to fuss over spacing and semicolons, and I don't think any hard-and-fast rules about code formatting should be enforced. Rules that are useful most of the time are occasionally going to be less than useful, as the `noqa`s sprinkled throughout the codebase and the weird thing I had to do with `if` in #1345 can attest. And some rules have no discernible value at all, like this one about one versus two semicolons. This is why, although I too use Emacs, I don't use auto-indent.

It's not hard for me to add a semicolon here, and one versus two semicolons makes no difference to me. But I don't want to spend time editing every PR for style issues that are more about tradition and fussy editors than about human legibility.